lib/internal/errors.js

```diff
@@ -1694,6 +1694,9 @@ E('ERR_UNKNOWN_FILE_EXTENSION', (ext, path, suggestion) => {
 }, TypeError);
 E('ERR_UNKNOWN_MODULE_FORMAT', 'Unknown module format: %s for URL %s',
   RangeError);
+E('ERR_UNSUPPORTED_MODULE_FORMAT',
+  'Module format %s for URL %s has no installed translator. You can use %s to add support for it.',
+  RangeError);
```

Comment:
Would it be better if we used TypeError?

Reply:
We were wondering about that